PurchasesCoreSwift/Logging/Strings/Emoji.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  Emoji.swift
+//  PurchasesCoreSwift
+//
+//  Created by Tina Nguyen on 12/8/20.
+//  Copyright © 2020 Purchases. All rights reserved.
+//
+
+import Foundation
+
+enum Emoji: String {
+    case apple = "\u{1F34E}\u{FE0F}"
```

Comment:
this should probably have occurred to me sooner, but can't you just use the emoji instead of the unicode? @tiinanguyen

Reply:
@aboedo Ah, yeah emojis can be used here instead of the unicode. I did a test run just to be sure. Can I just push a commit into `develop` without opening a PR?

Reply:
a PR is always better, even if only for a sanity check. I know it's a hassle, but it should just take a min, and it wouldn't block any of the other PRs